src/components/SideNav.tsx

```diff
@@ -167,10 +167,12 @@ const SideNav = ({ path }: SideNavProps) => {
       display={{ base: "none", lg: "block" }}
       aria-label={t("common:nav-developers-docs")}
     >
-      {docLinks.map((item, idx) => (
-        <NavLink item={item} path={path} key={idx} isTopLevel />
-      ))}
-    </Box>
+      <AccordionItem>
```

Comment:
This does not change much from what we currently have. It just wraps the current logic into an `Accordion` & `AccordionItem` components that doesn't add anything new.

The idea of using the `Accordion` component was to see if we could replace and simplify the current custom logic we have (most of which lives in the `NavLink` component). And, at the same time, it could also inherit the styles already applied to meet the DS specifications. You can see the current styles [here](https://dev--63b7ea99632763723c7f4d6b.chromatic.com/?path=/story/molecules-disclosure-content-accordions--basic).

Some quick pseudo code
```tsx
<Accordion allowToggle>
  <AccordionItem>
    <h2>
      <AccordionButton>
        <Box as="span" flex="1" textAlign="left">
          Level 1 - 1
        </Box>
        <AccordionIcon />
      </AccordionButton>
    </h2>
    <AccordionPanel>
      <SideNavLink>Level 2 - link 1</SideNavLink>
      <SideNavLink>Level 2 - link 2</SideNavLink>
      <SideNavLink>Level 2 - link 3</SideNavLink>
    </AccordionPanel>
  </AccordionItem>
  <AccordionItem>
    <h2>
      <AccordionButton>
        <Box as="span" flex="1" textAlign="left">
          Level 1 - 2
        </Box>
        <AccordionIcon />
      </AccordionButton>
    </h2>
    <AccordionPanel>
      <SideNavLink>Level 2 - link 1</SideNavLink>
      <SideNavLink>Level 2 - link 2</SideNavLink>
      <SideNavLink>Level 2 - link 3</SideNavLink>

      <Accordion allowToggle>
        <AccordionItem>
          <h2>
            <AccordionButton>
              <Box as="span" flex="1" textAlign="left">
                Level 2 - nested accordion
              </Box>
              <AccordionIcon />
            </AccordionButton>
          </h2>
          <AccordionPanel>
            <SideNavLink>Level 2 - link 1</SideNavLink>
            <SideNavLink>Level 2 - link 2</SideNavLink>
            <SideNavLink>Level 2 - link 3</SideNavLink>
            
            .... // More nested accordions
          </AccordionPanel>
        </AccordionItem>
      </Accordion>
    </AccordionPanel>
  </AccordionItem>
</Accordion>
```

Reply:
@pettinarip sure. I'll try to do it!